components/Modals/Modal.tsx

```diff
@@ -36,7 +36,7 @@ export default function Modal({
           theme,
           {
             'px-5 sm:h-fit sm:w-[550px]': !wide,
-            'sm:w-[90%]': wide,
+            'sm:w-fit': wide,
```

Comment:
This looks good! Especially for later chapters where the solutions are blocks of code (i.e. https://saving-satoshi-git-fork-benalleng-help-pag-b528e8-savingsatoshi.vercel.app/en/chapters/chapter-6/in-out-5) 


![Screenshot from 2024-06-06 13-32-56](https://github.com/saving-satoshi/saving-satoshi/assets/1823216/72080f82-886e-4ead-b801-bb840f211695)